src/queries/UserQueries.ts

```diff
@@ -55,3 +55,36 @@ export const UPDATE_USER = gql`
     }
   }
 `;
+
+export const GET_USERS = gql`
+  query getUsers($offset: Int, $perPage: Int) {
```

Comment:
Na pewno nie potrzebujemy tyle info o użytkowniku w prostej tabelce. W GraphQL chodzi o to aby pobierać tylko konieczne dane